l10n_it_riba_sale_commission/models/account_move.py

```diff
@@ -10,6 +10,27 @@ class AccountMove(models.Model):
 
     no_commission = fields.Boolean(string="Without commissions")
 
+    def _get_reconciled_invoices_partials(self):
+        res = super()._get_reconciled_invoices_partials()
+        to_remove = []
+        for (
+            partial,
+            amount,
+            counterpart_line,
+        ) in res:
+            riba_mv_line = self.env["riba.distinta.move.line"].search(
+                [("move_line_id", "=", partial.debit_move_id.id)]
+            )
+            if riba_mv_line:
+                riba_type = riba_mv_line.riba_line_id.type
+                if (
+                    riba_mv_line.riba_line_id.due_date
+                    + timedelta(days=riba_mv_line.riba_line_id.config_id.safety_days)
+                    > date.today()
+                ) and riba_type == "sbf":
+                    to_remove.append((partial, amount, counterpart_line))
```

Comment:
Looking at this in the future could possibly be pretty confusing, could you please add a comment inside this method, wherever do you prefer, explaining why you are ignoring this lines? Thanks!